### ISSUE TITLE
Error when using --support-pkg pointing to local tarball

### ISSUE THREAD

**Issue by eliasdorneles** [beeware/briefcase#60]
I'm trying to run briefcase (android target) using `--support-pkg` to provide a locally built support package, with the command:

    python setup.py android --support-pkg ../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz 

I'm getting the following error:

```
 * Installing support package...
Support package: ../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz
error: Invalid URL '../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz': No schema supplied. Perhaps you meant http://../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz?
```

I tried providing absolute paths, but the result is the same.

**Comment by eliasdorneles** [comment]
Trying to track the problem a bit further, it seems like an issue with the pip library:

```
>>> import pip
>>> pip.__version__
'9.0.1'
>>> link = pip.index.Link('../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz')
>>> link
<Link ../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz>
>>> pip.download.unpack_url(link, '/tmp/supportdir', download_dir='/tmp/downloaddir')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/download.py", line 821, in unpack_url
    hashes=hashes
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/download.py", line 659, in unpack_http_url
    hashes)
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/download.py", line 853, in _download_http_url
    stream=True,
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/_vendor/requests/sessions.py", line 488, in get
    return self.request('GET', url, **kwargs)
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/download.py", line 386, in request
    return super(PipSession, self).request(method, url, *args, **kwargs)
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/_vendor/requests/sessions.py", line 461, in request
    prep = self.prepare_request(req)
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/_vendor/requests/sessions.py", line 394, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/_vendor/requests/models.py", line 294, in prepare
    self.prepare_url(url, params)
  File "/home/elias/.virtualenvs/sketchapp/lib/python3.5/site-packages/pip/_vendor/requests/models.py", line 354, in prepare_url
    raise MissingSchema(error)
pip._vendor.requests.exceptions.MissingSchema: Invalid URL '../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz': No schema supplied. Perhaps you meant http://../voc-dev/voc/dist/Python-3.4-Android-support.b5.tar.gz?
```

**Comment by freakboy3742** [comment]
This has been addressed (and is now tested!) in the 0.3 development version.
